README.md

```diff
@@ -16,3 +16,98 @@ https://github.com/dwyl/intellectual-property and raise an
 on GitHub if you require clarification!
 
 See: https://github.com/dwyl/contributing/issues/1 + https://github.com/dwyl/summer-2015/issues/27
+
+## Recommended Guidelines
+
+### Issues
+
+At dwyl we use _issues_ to log our user stories and technical tasks for a given project. User stories are higher level issues that describe certain functionalities of the application you're building. These user stories get broken down into smaller **_technical_** tasks.
+
+### Issues > Before You Start
+
+Before you begin any work on a dwyl project please open an issue on the repository that you wish to contribute to.
+
+There **_must_** be a **comment** from **at least one other person** on the issue acknowledging it and/or confirming the requirement/bug before work starts. This is necessary to avoid doing _unnecessary work_. e.g. someone may have already solved that problem or implemented a similar feature on a different project and can point you in the direction of sample code or the PO/Client may deem the issue/story to be lower priority than something else on the list. The recommended workflow is as follows:
+
+1. First read through the existing stories/issues for the project to familiarise yourself with the current "backlog".
+2. Think of the keywords associated with the story/issue you want to create.
+3. Perform a quick search through the existing issues for the project with the keywords you identified e.g: https://github.com/dwyl/time/issues?utf8=%E2%9C%93&q=is%3Aissue%20email if email was your keyword.
+4. Once you are (quite) sure there is no existing issue/story covering what you want to solve/create move on to creating ... (see: steps below)
+
+### Issues > Validation
+
+After creating an issue/story (_before you estimate the time required or start working on it_) get people's input/feedback to _confirm_ that the description of the story is clear, has enough detail and addresses a real need/issue.
+
+This may feel a bit "_bureaucratic_" but (_we find that on balance_) it makes for better remote working because less discussion is "lost" and means everyone can follow along regardless of their location.
+
+>There **_must_** be a **comment** from **at least one other person** on the issue acknowledging it and/or confirming the requirement/bug before work starts. This is necessary to avoid doing _unnecessary work_. e.g. someone may have already solved that problem or implemented a similar feature on a different project and can point you in the direction of sample code or the PO/Client may deem the issue/story to be lower priority than something else on the list.
```

Comment:
This is repeated from above. I don't think we should remove it above entirely (because people might just read that section and nothing else), but maybe shorten and re-word into one sentence above and link to validation?